include/rtc/rtcpreceivingsession.hpp

```diff
@@ -58,9 +66,10 @@ class RTC_CPP_EXPORT RtcpReceivingSession : public MediaHandler {
 	uint32_t mTransit = 0;			// relative trans time for prev pkt
 	uint32_t mJitter = 0;	
 	
-	uint64_t mSyncRTPTS, mSyncNTPTS;
+	SyncPTS mSyncPTS{0,0};
 
 	std::atomic<unsigned int> mRequestedBitrate = 0;
+	std::mutex mSyncMutex;
```

Comment:
I'm not a maintainer, but creating a mutex only for 2 longs seems a bit overkill. They can still be stored as 2 uint64_t, or rather std::atomic<uint64_t>, then you can build your structure in an atomic way in the getSyncPTS without a lock.